### PR TITLE
Adds some admin docs

### PR DESCRIPTION
**Pull request by orta**
This PR adds some docs for admins with an overview of some useful repos and does some minor cleanup on the README

### CODE REVIEW

File: docs/admin.md
Comment (masters3d) [inline docs/admin.md]: When you say latest PR; Do you mean oldest PR?

File: README.md
Comment (sandersn) [inline README.md]: :+1:

Now that we support only 2.8 upward, prettier has a fighting chance on the whole repo. Maybe next tsconf?

File: docs/admin.md
Comment (sandersn) [inline docs/admin.md]:
```suggestion
When on-call, your goal is to try keep on top of the many open PRs for DefinitelyTyped; they are categorized into 
```

I LOVE SEMI-COLONS

Comment (sandersn) [reply docs/admin.md]: but only in natural language

File: docs/admin.md
Comment (sandersn) [inline docs/admin.md]: at some point we should have the content of 'oh no modules' in here, plus links to the handbook. for now can you add a list like

* no exceptions in tslint.json
* `strict: true` or equivalent four (five?) settings in tsconfig.json
* no `esModuleInterop: true`, no matter how tempting
* make sure export default is actually default in the source; react-* packages get a pass on this.

Also maybe link to my slide deck: https://docs.google.com/presentation/d/1Q4xfZSY7d9yHhtxSyb-DE85fTXB38RyF3nnyVyvenwc/edit?usp=sharing

File: docs/admin.md
Comment (sandersn) [inline docs/admin.md]: These two paragraphs are a good start, although eventually I'd like to have a bit more about the philosophy &mdash; that this isn't a code review in the usual sense, but an assessment of category and severity, followed by delegating to the people who actually know how to review the code. Since the bot tags the PRs with category and owners, it's mainly an exercise in co-ordinating between humans, and filling in any places the bot currently misses.

Also, something about how the sliding scale of package popularity affects reviews. The maintainer should review changes to popular packages, and very popular packages &mdash; those with a name you recognise, roughly &mdash; should have signoffs from at least two maintainers. Conversely, it's fine to overlook problems with changes to unpopular packages; in many cases only the author is using them anyway.

**Review by sandersn — CHANGES_REQUESTED** [review]

**Comment by orta** [comment]
thanks bot

**Comment by orta** [comment]
Gave it another one-over 👍

File: docs/admin.md
Comment (orta) [reply docs/admin.md]: I agree - I've expanded it!

File: docs/admin.md
Comment (orta) [reply docs/admin.md]: No SEMIS

File: docs/admin.md
Comment (orta) [reply docs/admin.md]: Thanks!

File: docs/admin.md
Comment (sandersn) [inline docs/admin.md]:
```suggestion
- There are a handful of authors who have shipped a lot of high quality contributions who you can happily delegate to
```

File: docs/admin.md
Comment (sandersn) [inline docs/admin.md]: This is hard in two ways:

1. Anything that's not a strict addition is likely to break code; and some additions break code, for example if they're to a class with subclasses.
2. DT packages are supposed to match the source's major.minor, so to bump semver, breaking changes can only ship when updating the types to a new major version.

The result is that so far we've told people that the Right Way to consume DT packages is to pin them to an exact major.minor.patch. That's not in the README and it's not common practise, but I think those are the things to change.

File: README.md
Comment (orta) [reply README.md]: 👍

**Comment by orta** [comment]
Alright, I'm not gonna fight the bot - this is good for now 👍

**Pull request merged**
